languages/java/exercises/concept/inheritance/.docs/after.md

```diff
@@ -0,0 +1,55 @@
+## Inheritance
+
+Java supports inheritance as it's core functionality and then to achieve a lot of OOPs principles like abstraction using
+inheritance.
+
+A class can extend another class using `extends` keyword and can inherit from an interface using `implements` keywords.
+
+## Access Modifiers
+
+The access modifiers define rules for the member variables of a class about their access from another classes(or anywhere in
+the code).
+
+There are mainly three access modifiers:
+
+- private
+- public
+- protected
+
```

Comment:
missing the fourth type of modifier in Java, the default one, aka `package visibility`

Reply:
Makes sense. Just didn't want to overwhelm the reader. But now that I think about it, I think we should have it.